packages/ui-buttons/src/BaseButton/styles.ts

```diff
@@ -356,6 +356,42 @@ const generateStyle = (componentTheme, props, state) => {
 
       // @ts-expect-error ts-migrate(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
       ...sizeVariants[size].iconSVG
+    },
+
+    flex: {
```

Comment:
Please add the missing css props from view (sent google sheet on slack)

Reply:
done